proj/cmake/platform_macosx.cmake

```diff
@@ -28,6 +28,15 @@ list( APPEND SRC_SET_AUDIO_COCOA
 	${CINDER_SRC_DIR}/cinder/audio/cocoa/FileCoreAudio.cpp
 )
 
+list( APPEND SRC_SET_QTIME
+	${CINDER_SRC_DIR}/cinder/qtime/AvfUtils.mm
+	${CINDER_SRC_DIR}/cinder/qtime/AvfWriter.mm
+	${CINDER_SRC_DIR}/cinder/qtime/MovieWriter.cpp
+	${CINDER_SRC_DIR}/cinder/qtime/QuickTimeGlImplAvf.cpp
```

Comment:
I can remove it, but I was not sure how it is preferred, since most of these files under `CINDER_SOURCES_OBJCPP` are also listed in `libcinder_source_files.cmake`.